docs/install/upgrade.md

```diff
@@ -36,12 +40,30 @@ Coder container:
 docker-compose pull coder && docker-compose up -d coder
 ```
 
-## Via Kubernetes
+### Kubernetes
 
 See
 [Upgrading Coder via Helm](../install/kubernetes.md#upgrading-coder-via-helm).
 
-## Via Windows
+### Coder AMI on AWS
+
+1. Run the Coder installation script on the host:
+
+   ```shell
+   curl -L https://coder.com/install.sh | sh
+   ```
+
+   The script will unpack the new `coder` binary version over the one currently
+   installed.
+
+1. If you're running Coder as a system service, you can restart it with `systemctl`:
```

Comment:
If it's the AWS AMI I believe they will *definitely* be running as a system service and should carry out this step.

Reply:
🙌 awesome - thanks! reworded in https://github.com/coder/coder/pull/16973/commits/9c056404c973b0ce0dc16dae2ca07404906d518c